doc/pages/Manual.md

```diff
@@ -563,38 +560,38 @@ is `true`. This is evaluated just when the command is going to be run.
            | <bool>
 ```
 
-Filters are evaluated at a certain point in time, and should not be mistaken
+Filters are evaluated at a certain point in time and should not be mistaken
 with package formulas, which express requirements.
 
 The following are allowed in filters:
-- String, integer and boolean literals
+- String, integer, and Boolean literals
 - Idents
 - Parentheses
 - Logical operators (binary AND `&`, binary OR `|`, prefix, unary NOT `!`)
 - The unary operator `?` for detecting whether a value is defined. `?expr`
-  evaluates to `true` if `expr` evaluates to a defined value, and `false` if it
+  evaluates to `true`, if `expr` evaluates to a defined value, and `false` if it
   evaluates to an undefined value.
 - Binary relational operators (`=`, `!=`, `<`, `<=`, `>`, `>=`)
 
 The comparisons are done using [Version Order](#version-ordering), including for
 integers, which are treated as strings. Relational operators have a higher
```

Comment:
Are these integers treated as strings?
If so, let's write: "...including for integers that are treated as strings."

...or does the Version Order comparisons treat integers as strings?
"...including for integers, which it treats like strings."